src/pages/Login/Image/Image.js

```diff
@@ -1,9 +1,13 @@
 import React from 'react';
 
-const Image = ({ src, alt , imageWrapperStyle}) => {
+const Image = ({ src, alt , imageWrapperStyle, imageStyle }) => {
+  const doesImageStyleExist = () => {
+    return imageStyle;
+  };
```

Comment:
Please turn it to a one-line function since it is an arrow function with one line to return.